.github/workflows/staging-odfe-cli.yml

```diff
@@ -0,0 +1,54 @@
+name: Staging ODFE CLI
+
+on:
+#  schedule:
+#    - cron: '0 12 * * *'
+  repository_dispatch:
+    types: [staging-odfe-cli]
+
+jobs:
+
+  Test-ODFE-CLI:
+    runs-on: ubuntu-18.04
+    name: Test-ODFE-CLI
+    strategy:
+      matrix:
+        go-version: [1.14]
+    steps:
+      - name: Set up AWS Cred
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - uses: actions/checkout@v1
+      - name: Retrieve plugin tags
+        run: |
+          echo "p_tag_cli=$(jq '.["odfe-cli"].cutversion' ./bin/plugins.json |sed 's/"//g')" >> $GITHUB_ENV
```

Comment:
use plugins-info

Reply:
This seems simpler approach and as its not dependent on another script.

Reply:
Let's make it consistent since everything else is using that script.

Reply:
Made change to pull tag info from plugins-info